php.md

```diff
@@ -23,3 +23,33 @@ PHP
     *   [PHPCDC](https://github.com/sebastianbergmann/phpdcd)
     *   [SensioLabs Security Checker](https://github.com/sensiolabs/security-checker)
     *   [PHP Mess Detector](http://phpmd.org)
+
+In-house conventions
+--------------------
+
+Use `echo` shortcut syntax within HTML blocks. Variable should be surrounded by spaces and omit semicolons:
+
+```html
+<!-- good -->
+<p><?= $var ?></p>
+
+<!-- bad -->
+<p><?=$var?></p>
+<p><?= $var; ?></p>
+```
+
+---
```

Comment:
The convention has been to use number lists for easy reference, rather than separating like this.